### PR TITLE
Add note about cluster state diffs

### PR DESCRIPTION
**Pull request by ywelsch**
Mentions cluster state diffs in CS publishing docs.

### CODE REVIEW

**Review by jasontedor — APPROVED** [review]
LGTM.

File: docs/reference/modules/discovery/publishing.asciidoc [hunk 1]
```diff
@@ -34,6 +34,16 @@ which defaults to `90s`. If a node has still not successfully applied the
 cluster state update within this time then it is considered to have failed and
 is removed from the cluster.
 
+Cluster state updates are typically published as diffs to the previous cluster
+state, which helps in saving on network bandwidth as well as speeding up the
```
Comment (DaveCTurner) [inline docs/reference/modules/discovery/publishing.asciidoc]:
```suggestion
state, which reduces the time and network bandwidth needed to```

File: docs/reference/modules/discovery/publishing.asciidoc [hunk 2]
```diff
@@ -34,6 +34,16 @@ which defaults to `90s`. If a node has still not successfully applied the
...
+publishing of cluster state updates. For example, when updating the mappings
```
Comment (DaveCTurner) [inline docs/reference/modules/discovery/publishing.asciidoc]:
```suggestion
publish a cluster state update. For example, when updating the mappings
```

**Review by DaveCTurner — APPROVED** [review]
Suggested optional minor wording changes, LGTM either way.

**Pull request merged**